classes/class-connectors.php

```diff
@@ -93,7 +93,7 @@ public function load_connectors() {
 				continue;
 			}
 			if ( $class->is_dependency_satisfied() ) {
-				$classes[] = $class;
+				$classes[ $class->name ] = $class;
```

Comment:
This will mean that, unfortunately, third-party extension initialising classes without the class name as the key will not get the benefit of auto-filtering the Actions menu.